src/mrack/run.py

```diff
@@ -43,6 +46,58 @@
 logger = logging.getLogger(__name__)
 
 
+class BrokenInstallation(Exception):
+    """Broken installation Exception."""
+
+    pass
+
+
+def get_mrack_dir():
+    """Get the mrack installation dir or raise error."""
+    for prefix in site.PREFIXES + [site.USER_BASE]:
+        path = os.path.join(prefix, "share/mrack")
+        if os.path.isdir(path):
+            return path
+    raise BrokenInstallation()
+
+
+def get_config_file_path():
+    """Locate mrack.conf file."""
+    try:
+        return os.path.join(get_mrack_dir(), "mrack.conf")
+    except BrokenInstallation:
+        logger.warning("Can not load mrack.conf using Python package defaults.")
+        return os.path.join(os.path.dirname(__file__), "data/mrack.conf")
+
+
+def get_default_file_path(filename):
```

Comment:
Why not to work with key names directly? What is the benefit of using key names similar to some default values when you will get an error anyway if you specify a different filename when calling the method.

I.e. the method seems to me too complex without  a clear benefit.